modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PageMemoryImpl.java

```diff
@@ -283,7 +283,7 @@
     private long[] sizes;
 
     /** Memory metrics to track dirty pages count and page replace rate. */
-    private DataRegionMetricsImpl memMetrics;
+    private final DataRegionMetricsImpl memMetrics;
```

Comment:
We need to add an assertion on `memMetrics` since it cannot be `null` value

Reply:
I think this will be an unnecessary check. Because it cannot be null in prod files. If it will be null, we will get NPE much earlier. This is only possible in tests, e.g if you create a fake class through the mock.